azure-pipelines.yml

```diff
@@ -64,6 +64,7 @@ stages:
     - script: |
         set -xe
         sudo pip3 install enum34
+        sudo pip install psutil
```

Comment:
py2 is deprecated.
pip3 install

Reply:
Thanks, fixed this on https://github.com/sonic-net/sonic-host-services/pull/108